client/src/components/ShoppingLists/ShoppingLists.tsx

```diff
@@ -0,0 +1,50 @@
+import { Box, Grid, Typography } from '@material-ui/core';
+import * as React from 'react';
+import useStyles from './useStyles';
+import ShoppingList from '../../components/ShoppingList/ShoppingList';
+import AddIcon from '@material-ui/icons/Add';
+
+//These static images are temporary
+import furnitureImage from '../../Images/furniture.png';
+import clothesImage from '../../Images/clothes.png';
+
+export default function ShoppingLists(): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <Grid container justify="flex-start" alignItems="flex-start" direction="column" className={classes.root}>
+      <Typography variant="h6">
+        {' '}
+        <Box fontWeight={700}>My Shopping Lists</Box>
```

Comment:
Why Box instead of using Typography for text?

Reply:
Box has the fontWeight property but Typography does not